docs/dashboard/import-export.md

```diff
@@ -0,0 +1,29 @@
+# Altis Dashboard Import Export
+
+The Altis Dashboard allows you to perform data migrations between environments, as well as perform data exports for use in local development environments.
+
+As shown below, you can find the Import & Export features under the Data tab of the selected Altis Environment.
+
+![import export view](../assets/import-export-view.png)
+
+From here you can view and download previously taken Exports, displayed as a list, ordered by date. 
+
+You can also create a new Export of your Database and/or Assets. Typically this would be comprised of your WordPress Uploads directory, in addition to any other assets your application may be uploading to your Uploads directory.)
```

Comment:
You've been capitalising "Export" but I'm not sure it needs it other than the H1 on the page and the name of the feature (line 5)
Similarly, I don't think 'database' or 'assets' need to be capitalised.

"Typically this would be comprised of your WordPress Uploads directory, in addition to any other assets your application may be uploading to your Uploads directory.)"
maybe
"Typically, this will be your WordPress Uploads directory, consisting of you media uploads and any other assets your application store there.)"